legate/core/runtime.py

```diff
@@ -883,6 +886,24 @@ def __init__(self, core_library: CoreLib) -> None:
             tuple[int, tuple[ProjExpr, ...]], int
         ] = {}
 
+        self._max_pending_exceptions: int = int(
+            self._core_context.get_tunable(
+                legion.LEGATE_CORE_TUNABLE_MAX_PENDING_EXCEPTIONS,
+                ty.uint32,
+            )
+        )
+        self._precise_exception_trace: bool = (
+            self._max_pending_exceptions > 1
+            and bool(
+                self._core_context.get_tunable(
+                    legion.LEGATE_CORE_TUNABLE_PRECISE_EXCEPTION_TRACE,
+                    ty.uint32,
```

Comment:
not `ty.bool_` as with `LEGATE_CORE_TUNABLE_NCCL_NEEDS_BARRIER`?

Reply:
47eeb80d968239bc44f4c9a65b2473b369b25f6c